pkg/apis/security/v1alpha1/types.go

```diff
@@ -154,12 +154,15 @@ type NetworkPolicyPort struct {
 	// If not specified, this field defaults to TCP.
 	// +optional
 	Protocol *v1.Protocol `json:"protocol"`
-	// The port on the given protocol. This can either be a numerical
+	// The port on the given protocol. This can be either a numerical
 	// or named port on a Pod. If this field is not provided, this
 	// matches all port names and numbers.
-	// TODO: extend it to include Port Range.
 	// +optional
-	Port *intstr.IntOrString `json:"port"`
+	Port *intstr.IntOrString `json:"port,omitempty"`
+	// To defines the end of the port range, being the end included within the range.
```

Comment:
did you mean "EndPort defines" instead of "To defines"?

(other occurence as well)

Reply:
Oh my bad. Fixed.